app/src/main/res/values-el/strings.xml

```diff
@@ -318,9 +318,9 @@
 
     <!-- Strings displayed only on Google Playstore. Optional, but good to have -->
     <!-- App title has to have less than 50 characters. If you cannot squeeze it, just remove a part of it -->
-    <string name="app_title">ΑΠΛΗ ΣΥΛΛΟΓΗ Pro - Διαχείριση &amp; Επεξεργασία</string>
+    <string name="app_title">Απλή Συλλογή Pro - Διαχείριση &amp; Επεξεργασία</string>
     <!-- Short description has to have less than 80 chars -->
-    <string name="app_short_description">Περιηγηθείτε στις αναμνήσεις σας χωρίς διακοπές με αυτήν τη συλλογή φωτό &amp; βίντεο</string>
+    <string name="app_short_description">Περιηγηθείτε στις αναμνήσεις σας χωρίς διακοπές με αυτήν την εφαρμογή φωτό &amp; βίντεο</string>
```

Comment:
this description is too long, it takes 83 chars. The max is 80. Even with counting &amp; as 1. Make it shorter please